README.md

```diff
@@ -1 +1,284 @@
+[here](https://pkg.go.dev/github.com/lukwil/mpi-operator/pkg/apis/kubeflow/v1#MPIJobSpec).  However,
+some of these values are overridden by NNF software and not all configurable options have been
+tested.
+
+For a full understanding of the other optinos in an NNF Container Profile, see the nnf-sos
+[samples](https://github.com/NearNodeFlash/nnf-sos/blob/master/config/samples/nnf_v1alpha1_nnfcontainerprofile.yaml)
+and
+[examples](https://github.com/NearNodeFlash/nnf-sos/blob/master/config/examples/nnf_v1alpha1_nnfcontainerprofiles.yaml).
+
+## Creating a DW Container Workflow
+
+With a container image and an NNF Container Profile, we are now ready to create a Workflow.
+
+The workflow definition will include two DW Directives:
+
+1. `#DW jobdw` to create the storage defined in the profile
+2. `#DW container` to create the containers and map the storage/profile.
+
+First, we'll create the storage. We will be using GFS2 for the filesystem:
+
+```none
+#DW jobdw name=nnf-container-example-gfs2 type=gfs2 capacity=50GB
+```
+
+Then, define the container. Note the `DW_JOB_my_storage=nnf-container-example-gfs2` argument matches what is in the NNF Container Profile and maps it to the name of the GFS2 filesystem created in the DW Directive above.
+
+```none
+#DW container name=nnf-container-example profile=nnf-container-example-profile DW_JOB_my_storage=nnf-container-example-gfs2
+```
+
+## Deploy Example to Cluster
+
+**Note**: The Flux workload manager may take care of all or most of the steps in this section. This is
+the manual way of doing things. You may want to consult a Flux expert on how to drive a container
+workflow using Flux.
+
+With a working Kubernetes cluster, the previous examples can be put together and deployed on the system. The files in this repository have done that.
+
+### Create the Profile and Workflow
+
+Deploy the profile and create the workflow:
+
+```shell
+kubectl apply -f nnf-container-example.yaml
+```
+
+### Assign Nodes to the Workflow
+
+For container directives, compute nodes must be assigned to the workflow. The NNF software will trace
+the computes node back to their local NNF nodes and the container will be executed on those NNF
+nodes.
+
+For the `jobdw` directive that is included, we must define the servers (i.e. NNF nodes) and the computes.
+
+Update the `servers` and `computes` resources assigned to the workflow.
+
+**Note**: you must change
+the node names in the `allocation-*.yaml` files to match your system. `allocationCount` must match
+the number of compute nodes being targeted for that particular NNF node. In the example,
+`rabbit-node-1` is attached to `compute-node-1` and `compute-node-2`, etc.
+
+```shell
+kubectl patch --type merge --patch-file=allocation-servers.yaml servers nnf-container-example-0
+kubectl patch --type merge --patch-file=allocation-computes.yaml computes nnf-container-example
+```
+
+### Progress Workflow
+
+At this point, the workflow should be in `Proposal` state and ready:
```

Comment:
Talk about status=completed, rather than the ready state.
We added this status=completed because there were times when the ready state was actually confusing things, especially in regard to distinguishing between PreRun being [finished/ready/completed/done/behind us], and when the compute application is running.